# Avoid "Undefined symbols" such as "___unw_remove_find_dynamic_unwind_sections" since dprint 0.50.1
# See: https://github.com/Homebrew/homebrew-core/pull/228646#issuecomment-3025970609
# See: https://github.com/NixOS/nixpkgs/pull/422595#pullrequestreview-2988926796

--- .cargo/config.toml	2025-06-30 21:51:34.000000000 +0000
+++ .cargo/config.toml	2025-08-01 04:02:36.846692711 +0000
@@ -1,3 +1,9 @@
 # statically link the c runtime https://github.com/rust-lang/rust/issues/100874
 [target.'cfg(all(windows, target_env = "msvc"))']
 rustflags = ["-C", "target-feature=+crt-static"]
+
+[target.'cfg(all(target_vendor = "apple"))']
+rustflags = [
+  "-C", "link-arg=-undefined",
+  "-C", "link-arg=dynamic_lookup"
+]
